IOS/SwiftSeedProject/SwiftSeedProject/AppDelegate.swift

```diff
@@ -7,12 +7,14 @@
 //
 
 import UIKit
+import Dip
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
-
+    private let container = DependencyContainer.configure()
```

Comment:
I think this line should be inside `didFinishLaunchingWithOptions `. The composition root could grow considerably and should only be built if the app was initialized successfully. With the current code, you have no control over that.